cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-12-14 10:43
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(blank=True, null=True, upload_to=cinema.models.movie_img_file_path),
```

Comment:
Ensure that the `movie_img_file_path` function is defined in the `cinema.models` module. This function is necessary for the `upload_to` parameter of the `ImageField`. If it doesn't exist, you need to define it or import it correctly to avoid any import errors.